locales/base/translation.json

```diff
@@ -1682,6 +1682,10 @@
       "title": "Before you swap",
       "body": "Swapping these assets may result in an unfavorable exchange rate. If you lower the amount, you may get a better rate."
     },
+    "missingPriceImpactWarning": {
+      "title": "Before you swap",
+      "body": "The price impact of this swap couldn't be estimated. Proceed carefully."
```

Comment:
Maybe there's a better wording for this?
This message shouldn't be shown normally, but there are some weird cases where this could happen.

Reply:
could we check with Ellen or Nitya before this is merged?

Reply:
Yes, I'll check with Nitya!

Reply:
Updated text from [designs](https://www.figma.com/file/jRtYI8GiSvPAaClBnRLLpu/In-App-Swap?type=design&node-id=3870-2341&mode=design&t=qxsdpx6QJ35SdgJA-0) shared by nitya.